packages/core/test/unit/html/NodeProcessor.data.js

```diff
@@ -230,13 +230,13 @@ module.exports.PROCESS_MODAL_HEADER_EXPECTED = `
 
 module.exports.PROCESS_MODAL_TITLE = `
 <modal title="**Lorem ipsum dolor sit amet**">
-  Title attribute should be inserted as internal _header slot.
+  Title attribute should be inserted as internal header slot.
```

Comment:
just one nit: let's remove the `internal` wording too since its all the same now

Reply:
Have made the changes!